config/ose-placeholders.js

```diff
@@ -14,15 +14,15 @@ var placeholders = {
   "mongo.admin_auth.pass": "{{env.MONGODB_ADMIN_PASSWORD}}",
   "fhredis.host": "{{env.REDIS_SERVICE_SERVICE_HOST}}",
   "fhredis.port": "{{env.REDIS_SERVICE_SERVICE_PORT}}",
-  "fhmetrics.host": "fh-metrics-service",
+  "fhmetrics.host": "fh-metrics-service.{{env.MBAAS_NAMESPACE}}",
```

Comment:
Wondering why these have the namespace appended to them...do they get passed on to the cloud apps or anything like that?